.gitignore

```diff
@@ -103,3 +103,4 @@ local/
 # Documentation build
 _build
 
+
```

Comment:
🤔 extra newline?

Reply:
@sankha555 see to it :)

Reply:
I guess that happenee when adding some lines to the gitignore file and then removing them as requested by @yashshah2820 . I supose that extra new line must have crept in then 😅

Reply:
please remove the extra line from the file

Reply:
@yashshah2820 Done! The extraline has been removed.